addons/web/static/src/views/form/form_controller.scss

```diff
@@ -857,6 +857,16 @@
         padding-right: 0 !important;
     }
 
+    //specific for styling labels and checkboxes from product_views.xml
```

Comment:
Either this code should/could be used by other modules too (you should find a more generic class-name in this case), or it should be move in the `product` module.

Reply:
The latest commit removed the class in product_view.xml making the changes only in the web modules and making the changes in form_controller.scss generic to situations where we have custom labels.